test/cast_test.exs

```diff
@@ -39,6 +39,55 @@ defmodule OpenApiSpec.CastTest do
       end
     end
 
+    test "allOf" do
```

Comment:
I'm gonna move these to their own test file corresponding to their module once I'm done.